### PR TITLE
clido: add page

### PR DESCRIPTION
**Pull request by Oglo12**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
- Clido 1.4

### CODE REVIEW

File: pages/common/clido.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# clido
+
+> Save-state TODO app for the terminal.
+> For more info, visit the Clido Wiki: <https://codeberg.org/Oglo12/clido/wiki>.
```
Comment (acuteenvy) [inline pages/common/clido.md:4]: The documentation link should always be prefixed with "More information:".
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout

```suggestion
> More information: <https://codeberg.org/Oglo12/clido/wiki>.
```

File: pages/common/clido.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Open a list in a text editor:
+
+`clido edit {{texteditor}}`
+
+- Get your version number:
+
+`clido -v`
+
+- Get a list of commands:
```
Comment (acuteenvy) [inline pages/common/clido.md:34]:
```suggestion
- Show a list of commands:
```

File: pages/common/clido.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Toggle autowrite:
+
+`clido toggle-autowrite`
+
+- Open a list in a text editor:
+
+`clido edit {{texteditor}}`
+
+- Get your version number:
```
Comment (acuteenvy) [inline pages/common/clido.md:30]:
```suggestion
- Show the version:
```

File: pages/common/clido.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`clido --lists`
+
+- Toggle autowrite:
+
+`clido toggle-autowrite`
+
+- Open a list in a text editor:
+
+`clido edit {{texteditor}}`
```
Comment (acuteenvy) [inline pages/common/clido.md:28]: `snake_case` should be used for multi-word placeholders.
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming
```suggestion
`clido edit {{text_editor}}`
```

File: pages/common/clido.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`clido edit {{text_editor}}`
+
+- Show the version:
+
+`clido -v`
+
+- Show a list of commands:
+
+`clido -h`
```
Comment (kbdharun) [inline pages/common/clido.md:36]: Thanks for your contribution and welcome to tldr. I have a minor suggestion for this page.

> `Display version:` and `Display help:` are the generic descriptions we use across most of our pages.

> Also, if long option `--version` and `--help` is supported feel free to update it to be consistent with other examples.

```suggestion
- Display version:

`clido -v`

- Display help:

`clido -h`
```

**Review by einverne — APPROVED** [review]
Looks good to me after @kbdharun 's suggestion.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**